src/main/scss/components/_dialogs.scss

```diff
@@ -1,15 +1,17 @@
 $jenkins-dialog-padding: 1.3rem;
 
 .jenkins-dialog {
-  border-radius: 0.6rem;
+  border-radius: var(--form-input-border-radius);
   border: none;
+  background-color: var(--card-background);
```

Comment:
Previously the dialog relied on the OS' background colour, which meant that it looked fine in light mode but switching to dark gives an inconsistent dark grey. A default blue outline also appears around the dialog (indicating its focused), this doesn't feel great so I've removed it.

**Before**
<img width="585" alt="image" src="https://github.com/user-attachments/assets/97e4c80d-8893-48af-a126-3ee85862978e" />

**After**
<img width="582" alt="image" src="https://github.com/user-attachments/assets/859ae21e-b321-4801-94a0-8732552e86d6" />